client/src/components/Suggestion/Suggestion.jsx

```diff
@@ -244,15 +287,8 @@ class Suggestion extends React.Component {
         return (
             <SuggestionContainer index = {index}>
 
-                <ImageBox onMouseEnter={this.mouseEnter} onMouseLeave={this.mouseLeave}><Image src={suggestion.pictureURL} alt="picture"></Image><Circle hover={this.state.hover}><Heartsign onClick={this.addFavorite} favorite={this.state.favorite} border={1} stroke={"black"} stroke-width={0.8} hover ={this.state.hover}/></Circle></ImageBox>
+                <ImageBox onMouseEnter={this.mouseEnter} onMouseLeave={this.mouseLeave}><Image src={suggestion.pictureURL} alt="picture"></Image><Circle hover={this.state.hover?1:0}><Heartsign onClick={this.addFavorite} favorite={this.state.favorite?1:0} border={1} stroke={"black"} strokeWidth={0.8} hover ={this.state.hover?1:0}/></Circle></ImageBox>
                 <RoomType> 
```

Comment:
nice passing of styled-components props and clever use of 1/0 boolean

Reply:
I agree!